File: week06/week06-4.cpp
//leet3005. week06-4.py
int maxFrequencyElements(int* nums, int numsSize) {
    int a[101] ={};
    int best =0;
    for(int i=0;i<numsSize; i++){
        int now =nums[i];
        a[now]++;
        if(a[now]>best) best = a[now];
    }
    int ans=0;
    for(int i=0;i<numsSize; i++){
        int now =nums[i];
        if(a[now]==best) ans++;
    
    }
    return ans;
}

File: week06/week06-4.py
#leetcode 3005. week06-4.py
class Solution:
    def maxFrequencyElements(self, nums: List[int]) -> int:
        a=[0]*101
        best=0
        for now in nums:
            a[now]+=1
            if a[now]>best: best = a[now]

        ans=0
        for now in nums:
            if a[now]==best: ans += 1
        return ans